cornflow/tests/data/full_case_raw.json

```diff
@@ -0,0 +1,197 @@
+{
```

Comment:
I prefer to import the files directly from cornflow_client. cornfllow_client is a requirement in any case. This helps later in the rare cases where the schemas change: we only change them in cornflow_client. It has happened before and it was horrible searching for all libraries where this file was present.

Reply:
Ok, then I will change it to import both files from cornflow client.